src/main/java/uk/ac/ebi/eva/pipeline/parameters/validation/job/GenotypedVcfJobParametersValidator.java

```diff
@@ -52,6 +53,8 @@ private CompositeJobParametersValidator compositeJobParametersValidator(JobParam
 
         Boolean skipAnnotation = Boolean.valueOf(jobParameters.getString(JobParametersNames.ANNOTATION_SKIP));
         if (!skipAnnotation) {
+            jobParametersValidators.add(new InputStudyIdValidator());
```

Comment:
InputStudyId now has always a empty default value to avoid having three different states: null, empty, value. By having a empty value by default we reduce the problem so that if you only want to execute that step is not mandatory , but if you are using the job it is mandatory.